### ISSUE TITLE
Geckodriver on macOS shows as posix tar.

### ISSUE THREAD

**Issue by madelaney** [webdriverextensions/webdriverextensions-maven-plugin#24]
I was trying to use this plugin but I seem to have hit a snag. Using the POM stanza below, I running into an issue where the Gecko driver (`drivers/drivers/geckodriver-mac-64bit`). When I try to run my test, the plugin fails to load as `cannot execute binary file`. I ran `file` against the driver and I got the following returned:
```
file drivers/geckodriver-mac-64bit
drivers/geckodriver-mac-64bit: POSIX tar archive
```

pom.xml stanza
```xml
<plugin>
          <groupId>com.github.webdriverextensions</groupId>
          <artifactId>webdriverextensions-maven-plugin</artifactId>
          <version>3.0.2</version>
          <executions>
              <execution>
                  <goals>
                      <goal>install-drivers</goal>
                  </goals>
              </execution>
          </executions>
</plugin>
```

**Comment by andidev** [comment]
I'll look into it

**Comment by andidev** [comment]
Sorry for the time it took, had to refactor some code to make some sense. It should be fixed in release 3.1.0 which should be ready to use within a couple of hours.

**Comment by andidev** [comment]
Please use 3.1.1 instead of 3.1.0 since it contains an issue where updating drivers is not possible
